src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/ExportPhysical/ExportPhysical.tsx

```diff
@@ -31,8 +36,38 @@ const CloseButton = styled(IconButton)(({ theme }) => ({
   },
 }));
 
-const ExportPhysical = ({ handleClose }: Props): ReactElement<Props> => {
+const LabelButton = styled(Button)(({ theme }) => ({
+  width: '90%',
+  margin: theme.spacing(1, 0),
+}));
+
+const ExportPhysical: React.FC<Props> = ({
+  handleClose,
+  accountListId,
+}: Props) => {
   const { t } = useTranslation();
+  const [session] = useSession();
+
+  const [createExportedContacts] = useCreateExportedContactsMutation({
+    onCompleted: (data) => {
+      window.location.replace(
+        `${data.exportContacts}?access_token=${session?.user.token}`,
+      );
+      handleClose();
+    },
```

Comment:
You can't get this from the return value of `await createExportedContacts`? Seems like it would be better to group that code instead of having a callback elsewhere.

Reply:
Yeah looking at it now, that's probably better. I think I had issues originally when trying but it seems to work fine now. I'll use this way instead!